lib/auth/auth_with_roles.go

```diff
@@ -2283,10 +2283,8 @@ func (a *ServerWithRoles) WebSessions() types.WebSessionInterface {
 
 // Get returns the web session specified with req.
 func (r *webSessionsWithRoles) Get(ctx context.Context, req types.GetWebSessionRequest) (types.WebSession, error) {
-	if err := r.c.currentUserAction(req.User); err != nil {
```

Comment:
Was this check not needed?

Reply:
Only the Proxy Role needs to retrieve web sessions for the WebUI to function correctly, so yes the check can be removed.

Previously, this would allow users to retrieve their own web session (and secret private key / certs) directly using the `/webapi`, which could then be used to bypass hardware key requirements.